tests/test-translations.php

```diff
@@ -133,4 +133,18 @@ public function test_term_translations() {
 
 	}
 
+	public function test_canonical_content_fallback() {
+		$this->assertSame( 'en_US', get_locale() );
+
+		$en = $this->factory->post->create_and_get();
+		// In normal operation, the create_empty_translations method is called on an immediate single cron job
+		$GLOBALS['bbl_jobs']->create_empty_translations($en->ID);
+		$fr = bbl_get_post_in_lang( $en->ID, 'fr_FR', true );
+
+		// @FIXME: These tests fail due to the interaction of Babble_Post_Public::get_post_in_lang and Babble_Jobs::create_empty_translations
+		$this->assertSame( $en->post_title,   $fr->post_title );
```

Comment:
@simonwheatley looking through the code, this test seems to be expecting the title to default to the default post, however what I understand, there is only logic in place to fallback to the default lang if a post in that language does not exist. In this case, we do have a translated post in `fr_FR` so no "fallback" of content will happen. See https://github.com/Automattic/babble/blob/develop/class-post-public.php#L1337